src/simulators/matrix_product_state/matrix_product_state_internal.cpp

```diff
@@ -989,20 +1017,33 @@ complex_t MPS::expectation_value_pauli(const reg_t &qubits, const std::string &m
   return result;
 }
 
-std::ostream& MPS::print(std::ostream& out) const
-{
-	for(uint_t i=0; i<num_qubits_; i++)
+std::ostream& MPS::print(std::ostream& out) const {
+  for(uint_t i=0; i<num_qubits_; i++)
+    {
+      out << "Gamma [" << i << "] :" << std::endl;
+      q_reg_[i].print(out);
+      if(i < num_qubits_- 1)
 	{
-	  out << "Gamma [" << i << "] :" << std::endl;
-	  q_reg_[i].print(out);
-	  if(i < num_qubits_- 1)
-	    {
-	      out << "Lambda [" << i << "] (size = " << lambda_reg_[i].size() << "):" << std::endl;
-	      out << lambda_reg_[i] << std::endl;
-	    }
+	  out << "Lambda [" << i << "] (size = " << lambda_reg_[i].size() << "):" << std::endl;
+	  out << lambda_reg_[i] << std::endl;
 	}
-	out << std::endl;
-	return out;
+    }
+  out << std::endl;
+  return out;
+}
+
+std::ostream& MPS::sort_and_print(std::ostream& out) {
```

Comment:
Is this function used? I'd remove it if it's not.
If you want to keep it, I'd strongly recommend removing the print part as we already have a print function, and changing the signature to: `void MPS::sort_qubits()`.

Reply:
Removed this function.